### PR TITLE
Fix Criteria's orderBy example

### PR DESCRIPTION
**Pull request by malarzm**
This was reported in https://github.com/doctrine/collections/issues/148

### CODE REVIEW

File: docs/en/reference/query-builder.rst [hunk 1]
```diff
@@ -505,7 +505,7 @@ using ``addCriteria``:
     // ...
 
     $criteria = Criteria::create()
-        ->orderBy(['firstName', 'ASC']);
+        ->orderBy(['firstName' => 'ASC']);
```
Comment (Ocramius) [inline docs/en/reference/query-builder.rst]: Can you please use `Criteria::ASC` here?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: docs/en/reference/query-builder.rst (hunk 1 above)
Comment (malarzm) [reply docs/en/reference/query-builder.rst]: Sure, gimme a moment though as now I need to clone :D

Comment (Ocramius) [reply docs/en/reference/query-builder.rst]: Edit your branch from the online editor :D

Comment (malarzm) [reply docs/en/reference/query-builder.rst]: And clone later because of squash request? :P

Comment (Ocramius) [reply docs/en/reference/query-builder.rst]: What squash request? I'm not THAT mad.

**Review by Ocramius — APPROVED** [review]

File: docs/en/reference/query-builder.rst (hunk 1 above)
Comment (malarzm) [reply docs/en/reference/query-builder.rst]: I'd normally ask for one ;)

**Pull request merged**